src/main/scala/kinesis/mock/cache/CacheConfig.scala

```diff
@@ -31,6 +35,28 @@ final case class CacheConfig(
 object CacheConfig {
   implicit val cacheConfigReader: ConfigReader[CacheConfig] = deriveReader
   implicit val cacheConfigCirceEncoder: Encoder[CacheConfig] = deriveEncoder
+
+  implicit val initializeStreamsReader
+      : ConfigReader[List[CreateStreamRequest]] = {
+    ConfigReader.fromString { s =>
+      s.split(',')
+        .toList
+        .map(_.split(':').toList)
+        .traverse {
+          case name :: count :: Nil =>
```

Comment:
```
scala> ":someStream".split(":")
val res0: Array[String] = Array("", someStream)
```
Looks like you'll want to check if the `name` is empty.

Reply:
yea i just caught that, merged now